__tests__/app.test.js

```diff
@@ -257,3 +257,94 @@ describe("POST /api/articles/:article_id/comments", () => {
     })
 })
 
+describe("PATCH /api/articles/:article_id", () => {
+    test("returns updated article", () => {
+        return request(app).patch('/api/articles/1')
+        .send({
+            inc_votes: 22
+        })
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toMatchObject(
+                {
+                    title: "Living in the shadow of a great man",
+                    article_id: 1,
```

Comment:
We know it will have other properties on it - you could put these on and just have the value as string or number etc